src/AppInstallerRepositoryCore/Rest/RestSourceFactory.cpp

```diff
@@ -18,11 +18,15 @@ namespace AppInstaller::Repository::Rest
 		{
 			std::shared_ptr<ISource> Create(const SourceDetails& details, IProgressCallback& progress) override final
 			{
+				UNREFERENCED_PARAMETER(progress);

```

Comment:
If you are not going to use it ever, just remove the name from the function signature.